docs/requirements.txt

```diff
@@ -179,7 +179,7 @@ platformdirs==3.0.0
     # via jupyter-core
 psutil==5.9.4
     # via distributed
-psycopg2==2.9.5
+#psycopg2==2.9.5
```

Comment:
This is a generated file so the modification will be lost the next time the file is generated.